app/models/additional_email.rb

```diff
@@ -1,32 +1,39 @@
 class AdditionalEmail < ActiveRecord::Base
   include ModelsModule
-  attr_accessible :user_id, :email, :confirmed_at
+
+  THOUGHTWORKS_EMAIL = "thoughtworks.com"
   VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
+
+  devise :database_authenticatable, :confirmable
+
+  attr_accessible :user_id, :email, :confirmed_at
+
+  belongs_to :user
+
   validates :email, presence:   true,
                     format:     { with: VALID_EMAIL_REGEX },
                     uniqueness: { case_sensitive: false }
   validates :user_id, presence: true
-  validate :must_be_thoughtworks_email
-  validate :uniqueness_of_email
-  devise :database_authenticatable, :confirmable
-  belongs_to :user
+  validate  :must_be_thoughtworks_email
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.